### PR TITLE
Feat: Autocomplete paths

### PR DESCRIPTION
**Pull request by chottuthejimmy**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -107,8 +107,22 @@ let runID = new Date().getTime();
...
-  completions.concat(tasks)
+  let filesAndFolders = fs.readdirSync('./')
+  filesAndFolders = filesAndFolders.map(file => {
+    let stats = fs.statSync(file)
+    if(stats.isDirectory()){
+      return file + '/' // add a '/' at the end of the folder name
+    } else {
+      return file
+    }
+  })
```
Comment (ianjennings) [inline index.js:118]: this should be nested under the conditional on line 120

File: index.js [hunk 2]
```diff
@@ -107,8 +107,22 @@ let runID = new Date().getTime();
...
+
+  if (line.startsWith('/run ')){
+    var partialName = line.substring(5) // works only when there is a space after /run
+    var matches = filesAndFolders.filter(name => name.startsWith(partialName))
+    var fullCommand = matches.map(name => '/run ' + name)
```
Comment (ianjennings) [inline index.js:123]: use `let` instead of `var`

File: lib/init.js [hunk 3]
```diff
@@ -61,7 +61,7 @@ module.exports = async () => {
     console.log('')
     console.log(chalk.dim('This is a preview of the Testdriver.ai'));
     console.log(chalk.dim('Please report any issues in our Discord server: '));
-    console.log(chalk.dim('https://discord.gg/3Q3QkZ2'));
+    console.log(chalk.dim('https://discord.gg/22RdN6tp'));
```
Comment (ianjennings) [inline lib/init.js:64]: why keep changing?

**Review by ianjennings — CHANGES_REQUESTED** [review]
Biggest thing is `let` vs `var`

File: index.js (hunk 1 above)
Comment (chottuthejimmy) [reply index.js:118]: ya, I would have to move and define a recursive call for this to handle the subdirectories and will later call it inside line 120

File: index.js (hunk 2 above)
Comment (chottuthejimmy) [reply index.js:123]: Oh ok got it. 
I did see one var [here](https://github.com/testdriverai/testdriverai/blob/3d510b1b227411d4379c61f3275c82252fdf1d44/index.js#L106)

File: lib/init.js (hunk 3 above)
Comment (chottuthejimmy) [reply lib/init.js:64]: ya so some of these links are configured to expire after 7 days, I am just fixing the broken ones

**Pull request closed without merging**